src/overmapbuffer.cpp

```diff
@@ -1907,7 +1907,7 @@ std::vector<point_abs_om> overmapbuffer::find_highway_intersection_bounds( const
 }
 
 
-std::string overmapbuffer::get_description_at( const tripoint_abs_sm &where )
+std::string overmapbuffer::get_description_at( const tripoint_abs_sm &where, bool draw_origin )
```

Comment:
I would prefer if you avoided calling this function. It allocates a new string each time it is called, which is generally a bad idea in the ui code. One garbage string every frame won’t ruin us, but it adds up. Just write a function that directly uses ImGui to print out the information without manipulating strings.